app/inputs/multi_value_with_help_input.rb

```diff
@@ -1,6 +1,4 @@
 class MultiValueWithHelpInput < MultiValueInput
```

Comment:
Probably not. I am currently investigating what views in CC and Sufia need to be altered when we remove these classes.